package.json

```diff
@@ -32,7 +32,7 @@
     "create-component": "node scripts/create-component.js",
     "create-package": "node scripts/create-package.js",
     "create-page": "node scripts/create-page.js",
-    "checkchange": "node common/scripts/install-run-rush.js change -v",
+    "checkchange": "node common/scripts/install-run-rush.js change -v -b origin/6.0",
```

Comment:
Can this go in separately? (or is it already going in separately in another branch?)

Also wonder if we should introduce something similar to the `change` command against `upstream/6.0`.

Reply:
JD asked me to test this. It could be separated but I'd need to merge it in just to get CI running on this PR. I'm also all for separating PR concerns but this is just a one liner that can be pulled out or changed pretty easily.

Reply:
Not sure how long this PR will be in flight so I made #9453.

Reply:
Thanks! To clarify, I'm fine with having it here to unblock your PR's CI build, just also wanted it checked in sooner to unblock others.